apps/linode-marketplace-wazuh/roles/post/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+-
+# motd and creds gen
+- name: copy MOTD template to /etc/motd
+  template:
+    src: templates/motd.j2
+    dest: /etc/motd
+
+- name: writing wazuh creds into file
+  copy:
+    dest: /home/{{ username }}/.credentials
+    owner: "{{ username }}"
+    group: "{{ username }}"
+    mode: '0600'
+    content: |
+      Sudo Username: {{ username }}
+      Sudo Password: {{ password }}
+
+- name: Read content from /tmp/wazuh-install-files/wazuh-passwords.txt
+  ansible.builtin.slurp:
+    src: /tmp/wazuh-install-files/wazuh-passwords.txt
+  register: wazuh_passwords_content
+
+- name: decode and append wazuh_passwords_content to creds file
+  ansible.builtin.shell:
+    cmd: |
+      echo "" >> /home/{{ username }}/.credentials
+      echo "{{ wazuh_passwords_content['content'] | b64decode }}" >> /home/{{ username }}/.credentials
```

Comment:
This needs to be reworked a bit to avoid using `shell`. We can use `lineinfile` to insert before EOF